### ISSUE TITLE
Course, May 2023

### ISSUE THREAD

**Issue by mvsjober** [csc-training/intro-to-dl#10]
Recurring TODOs:
- [x] Reserve Dogmi
- [x] Add to course scheduling
- [x] JIRA ticket
- [x] Fill course form
- [x] Eventilla page
- [x] Create course project for Puhti
- [x] Create course project for LUMI
- [x] GPU reservation(s)
- [x] Notebooks course request (Mats)
- [x] Update/review lectures (Mats)
- [x] Update/review lectures (Markus)
- [x] Test Notebooks + exercises
- [x] Test Puhti exercises
- [x] Test LUMI exercises
- [x] Fix project in Puhti/LUMI exercises
- [x] Test day1 with Google colab
- [x] Create Zoom room
- [x] HedgeDoc/HackMD pages
- [x] HedgeDoc sub pages for discussion, day 1 exercises, day 2 exercises, passwords
- [x] Copy data from old to new project in Puhti/LUMI
- [x] Ask Mikael Kanerva about/reserve wireless microphones for Dogmi, NOTE: needed in Debatti 3.5. kl 16:30
- [x] Go through last time's comments and add TODOs here if needed
- [x] Print signs for LSC lobby and Dogmi

Course advertising:
- [x] Advertise course in EuroCC
- [x] CSC newsletter
- [x] Linkedin
- [x] CSChpc Twitter
- [x] Weekly user meeting
- [x] docs pages (ML user guide, pytorch, tensorflow)
- [x] LUMI users mailing list

Specific to this instance:
- [x] Add mention of hybrid status/remote option to the eventilla page
- [ ] ~Consider using NVME in all scripts?~
- [x] Convolutions lecture update? ConvNeXt et al?
- [ ] ~CVSLogger in exercises~
- [x] Diffusion models exercise: try with traffic signs
- [x] update exercises back to using web interface
- [ ] add as extracuricular to Ex6: https://keras.io/guides/keras_cv/generate_images_with_stable_diffusion/
 
TODOs from last time's comments:
- [ ] ~Take a look at PyTorch vs Keras accuracy differences in CNN/RNN 20ng exercises.~
- [x]  "When to show CNN applications slides? Maybe on day 1 after all?"
- [x] "Day 2 morning has too much talk by me, and then there essentially 10 mins for actually doing exercises. Maybe skip the frameworks part in the beginning?"
- [ ] Matias: "Tables in exercises: each model could have their own so it would be easier to compare"

After registration has closed:
- [x] Request Notebooks training accounts
- [x] Send email to participants [draft here](https://siili.rahtiapp.fi/PDL-welcome-email)
- [x] check new registrations and send email (Friday 28.4 after 12)
- [x] check new registrations and send email (Tuesday 2.5.)

During course:
- [x] Remember to record lectures!!

After course:
- [x] Post lecture recordings
- [x] Send attendance info to Events team (for certificates)
- [ ] Move any new git commits to mls

**Comment by mvsjober** [comment]
Eventilla page: https://ssl.eventilla.com/event/8aPek

**Comment by mvsjober** [comment]
Mirror in HackMD in case of HedgeDoc/siili problems: https://hackmd.io/OIAaKnvHTiy44F_NtLG9tg

**Comment by mvsjober** [comment]
Lecture 2 went a bit over time, left 15 mins for exercise...

**Comment by mvsjober** [comment]
Second day morning still a bit of a mess, switching between slides exercise explanation etc. Need a clearer script.

**Comment by avancra** [comment]
Here are some notes I took during the course:
- Test tensor board on LUMI and update instructions in the day 2 README
- Add a note about "pip install –user tensorflow-hub" for exercise 5
- Possibly idea: make a dedicated PowerPoint for exercise slides
- Make a bigger reservation on Puhti
- Create a mailing list for course organisers so everyone can help in case of issue received by email
- Do we propose solutions for Day 2 exercises?
